.github/workflows/update-bbb-recording-player.yml

```
@@ -0,0 +1,36 @@
+          fetch-depth: 2
+      - name: Extract the current BBB Recording Player version from Docker file
+        run: echo "CURRENT_PLAYER_VERSION=$(sed -n 's/ARG PLAYBACK_PLAYER_VERSION=//p' docker/app/Dockerfile)" >> $GITHUB_ENV
+      - name: Compare the versions
+        run: echo "NEEDS_UPDATE=$(dpkg --compare-versions ${{ env.LATEST_PLAYER_VERSION }} gt ${{ env.CURRENT_PLAYER_VERSION }} && echo true || echo false)" >> $GITHUB_ENV
+      - name: Update the BBB Player version
+        if: ${{ env.NEEDS_UPDATE == 'true' }}
+        run: |
+          sed -i "s/ARG PLAYBACK_PLAYER_VERSION=.*/ARG PLAYBACK_PLAYER_VERSION=${LATEST_PLAYER_VERSION}/" docker/app/Dockerfile
+      - name: Create PR with the update
+        if: ${{ env.NEEDS_UPDATE == 'true' }}
+        uses: peter-evans/create-pull-request@v3
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)